optimum/exporters/onnx/model_configs.py

```diff
@@ -745,3 +746,43 @@ def generate_dummy_inputs(self, framework: str = "pt", **kwargs):
 class WhisperOnnxConfig(AudioToTextOnnxConfig):
     NORMALIZED_CONFIG_CLASS = NormalizedSeq2SeqConfig
     ATOL_FOR_VALIDATION = 1e-3
+
+
+class MobileNetV1OnnxConfig(VisionOnnxConfig):
+    NORMALIZED_CONFIG_CLASS = NormalizedVisionConfig
+    MIN_TORCH_VERSION = version.parse("1.11")
+
+    @property
+    def inputs(self) -> Mapping[str, Mapping[int, str]]:
+        return OrderedDict([("pixel_values", {0: "batch"})])
+
+    @property
+    def outputs(self) -> Mapping[str, Mapping[int, str]]:
+        if self.task == "image-classification":
+            return OrderedDict([("logits", {0: "batch"})])
+        else:
+            return OrderedDict([("last_hidden_state", {0: "batch"}), ("pooler_output", {0: "batch"})])
```

Comment:
Can be removed

Reply:
the `inputs` is defined only here, so should be kept no? And for the outputs there's the `pooler_output` which is not handled in the base.py `_TASK_TO_COMMON_OUTPUTS`